### PR TITLE
PFM-ISSUE-11835: Improve merge-skeleton command

### PR DESCRIPTION
**Pull request by vladimir-arsov**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cplace/cli",
-  "version": "0.19.19",
+  "version": "0.19.20",
```
Comment (Kaltenbach) [inline package.json:3]: We will need to join our two updates, see #114

File: src/commands/repos/MergeSkeleton.ts [hunk 2]
```diff
@@ -41,53 +42,47 @@ export class MergeSkeleton extends AbstractReposCommand {
...
     protected targetBranchIsTracked: boolean = false;
-    protected baseBranch: string = '';
+    protected baseBranch: string;
     protected status: IGitStatus;
     protected mergeSuccess: boolean = true;
 
     public async execute(): Promise<void> {
         const pathToRepo = path.join(process.cwd());
         const repo = new Repository(pathToRepo);
+        this.mergeSuccess = true;
```
Comment (Kaltenbach) [inline src/commands/repos/MergeSkeleton.ts:52]: Aint this already true as it is set above (L47)?

**Review by Kaltenbach — CHANGES_REQUESTED** [review]
just a small question about initialisation.

File: src/commands/repos/MergeSkeleton.ts (hunk 2 above)
Comment (vladimir-arsov) [reply src/commands/repos/MergeSkeleton.ts:52]: yes, this seems to be obsolete

**Review by Kaltenbach — APPROVED** [review]

**Pull request merged**